src/components/config/index.ts

```diff
@@ -161,9 +162,26 @@ export const createConfig = async (flags: any) => {
     open('https://hyperjumptech.github.io/monika-config-generator/')
   } else {
     const { path, type } = getPathAndTypeFromFlag(flags)
+
+    if (!existsSync(path)) {
+      log.error(`Couldn't found the ${path} file.`)
+      return
+    }
+
     const parsed = parseConfig(path, type)
     const file = flags.output || 'monika.json'
 
+    if (existsSync(file) && !flags.force) {
+      const ans = await cli.prompt(
+        `\n${file} file is already exists. Overwrite (Y/n)?`
+      )
+
+      if (ans.toLowerCase() !== 'y') {
+        log.warn(`Exit without overwriting ${file}.`)
```

Comment:
Kang Mus maybe better for the exit message to be: 
Command cancelled. You can use the `-o` flag to specify an output file or `--force` to overwrite without prompting

Reply:
done